themes/gatsby-theme-deriv/src/features/components/quill/videoModal/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback } from 'react'
+import ModalVideo from 'react-modal-video'
+import 'react-modal-video/scss/modal-video.scss'
```

Comment:
move it to root would be good. Otherwise this css will be loaded everytime this component loads.